$MirOS: ports/misc/mc/patches/patch-slang_slsignal_c,v 1.1 2005/03/18 15:50:27 tg Exp $
$OpenBSD: patch-slang_slsignal_c,v 1.1 2004/04/11 00:40:59 naddy Exp $
--- slang/slsignal.c.orig	Mon Oct  7 04:40:41 2002
+++ slang/slsignal.c	Fri Oct 15 17:49:34 2004
@@ -226,7 +226,7 @@ int SLsystem (char *cmd)
 	(void) sigprocmask (SIG_SETMASK, &save_mask, NULL);
 # endif
 
-	execl ("/bin/sh", "sh", "-c", cmd, NULL);
+	execl ("/bin/sh", "sh", "-c", cmd, (char *)NULL);
 	_exit (127);
      }
    else
